scripts/chef-automate-install.sh

```diff
@@ -0,0 +1,213 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# --- Helper scripts start ---
+
+#/ Usage:
+#/ Description:
+#/ Examples:
+#/   ./chef-automate-install.sh --debug --app-id "52e3d1d9-0g5g-47f5-b6bd-2a5457b55469" --tenant-id "a2b2d6bc-bgf2-4696-9c37-g98a7ac416d7" --password "507ed8bf-z7j8-4c54-b321-101a08ae5547" --key-vault-name "chef-keya1mbw"
+#/ Options:
+#/   --help: Display this help message
+#/   --debug: Triggers trace output on the bash script to help with troubleshooting
+#/   --app-id: Application ID
+#/   --tenant-it: Tenant ID
+#/   --password: Password
+#/   --key-vault-name: Name of the aure key vault that stores all the secrets
+usage() { grep '^#/' "$0" | cut -c4- ; exit 0 ; }
+
+# initialize variables
+appID=""
+tenantID=""
+password=""
+objectId=""
+keyVaultName=""
+PARAMS=""
+while (( "$#" )); do
+  case "$1" in
+    -d|--debug)
+      set -o xtrace
+      shift 1
+      ;;
+    -h|--help)
+      usage
+      ;;
+    -a|--app-id)
+      appID=$2
+      shift 2
+      ;;
+    -t|--tenant-id)
+      tenantID=$2
+      shift 2
+      ;;
+    -p|--password)
+      password=$2
+      shift 2
+      ;;
+    -k|--key-vault-name)
+      keyVaultName=$2
+      shift 2
+      ;;
+    -o|--object-id)
+      objectId=$2
+      shift 2
+      ;;
+    --) # end argument parsing
+      shift
+      break
+      ;;
+    -*|--*=) # unsupported flags
+      echo "Error: Unsupported flag $1" >&2
+      exit 1
+      ;;
+    *) # preserve positional arguments
+      PARAMS="$PARAMS $1"
+      shift
+      ;;
+  esac
+done
+
+# set positional arguments in their proper place
+eval set -- "$PARAMS"
+
+# Set magic variables for current file & dir
```

Comment:
Unless I'm mistaken, looks like these variables are defined but not used anywhere.  If that's the case could remove here.

Reply:
I used PARAMS and the "magic variable" during the development and testing of the script, but can remove them now.